src/test/resources/hudson/plugins/ec2/Mac.yml

```diff
@@ -9,6 +9,6 @@ jenkins:
           - description:
             ami: "ami-5678"
             labelString: "mac clear"
-            type: "Mac1Metal"
+            type: "mac1.metal"
```

Comment:
Is this a breaking change?

Reply:
@res0nance No, it is not. Old values are still accepted.